ff/ff_derive/src/lib.rs

```diff
@@ -1062,9 +1058,11 @@ fn prime_field_impl(
                 ret
             }
 
-            fn inverse(&self) -> Option<Self> {
+            /// WARNING: THIS IS NOT ACTUALLY CONSTANT TIME YET!
+            /// TODO: Make this constant-time.
```

Comment:
This will likely require inlining addition-chain finding into the proc macro.

Reply:
That sounds beautiful! If only I could stand how many dependencies proc-macro wants to bring in.

Reply:
Don't need to worry about proc-macro dependencies if your crate itself is a procedural macro 🙃

Reply:
Opened https://github.com/zkcrypto/ff/issues/15 for this.